smarts/sstudio/types.py

```diff
@@ -513,29 +513,6 @@ def resolve_offset(offset, geometry_length, lane_length):
             else:
                 return float(offset)
 
-        def pick_remaining_shape_after_split(geometry_collection, length, lane):
-            lane_shape = geometry_collection
-            if not isinstance(lane_shape, GeometryCollection):
-                return lane_shape
-
-            # For simplicty, we only deal w/ the == 1 or 2 case
-            if len(lane_shape) not in {1, 2}:
-                return None
-
-            if len(lane_shape) == 1:
-                return lane_shape[0]
-
-            expected_bbox_area = lane.getWidth() * length
-            keep_index = 0
-            if (lane_shape[0].minimum_rotated_rectangle.area - expected_bbox_area) < (
-                lane_shape[1].minimum_rotated_rectangle.area - expected_bbox_area
```

Comment:
I didn't really understand how this area comparison works...but it is not selecting the correct index in 4lane_t.

Reply:
Yes this was guessing that the larger of the two geometries was most correct and seems to have resulted in the wrong behaviour. The change might work but it will need to be tested with a box cut in another direction.